### PR TITLE
btrfs: remove C constant redefinitions

### LINKED ISSUE #15: btrfs.go:103: constant 18446744073709551616 overflows C.__u64
With: 
-   github.com/containerd/btrfs 72c0a3512e5703f4f03f392167117122b550b6e8
-   btrfs-progs-devel.x86_64 4.14.1-1.fc28                                                                         
-   golang-github-pkg-errors-devel.noarch 0.7.1-0.5.gita887431.fc27               
-   btrfs-progs.x86_64 4.14.1-1.fc28                                              
-   e2fsprogs-libs.x86_64 1.43.8-1.fc28 

                                          
       
Tests fail with:
```shell
 + go test -buildmode pie -compiler gc -ldflags ' -X github.com/containerd/btrfs/version.commit=72c0a3512e5703f4f03f392167117122b550b6e8 -X github.com/containerd/btrfs/version=0 -extldflags '\''-Wl,-z,relro  -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '\'''
 # github.com/containerd/btrfs
 ./btrfs.go:103: constant 18446744073709551616 overflows C.__u64
 ./btrfs.go:104: constant 18446744073709551616 overflows C.__u64
 ./btrfs.go:105: constant 18446744073709551616 overflows C.__u64
```

### PR DESCRIPTION
**Pull request by stevvooe**
Signed-off-by: Stephen J Day <stephen.day@docker.com>

Closes #15

### CODE REVIEW

**Comment by stevvooe** [comment]
Wait for confirmation from reporters before merging.

**Comment by dmcgowan** [comment]
> Wait for confirmation from reporters before merging.

Agreed, LGTM

**Review by justincormack — APPROVED** [review]

**Pull request merged**
